x11-wm/awesome/awesome-4.0.ebuild

```diff
@@ -0,0 +1,123 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI="6"
+inherit cmake-utils eutils pax-utils
+
+DESCRIPTION="A dynamic floating and tiling window manager"
+HOMEPAGE="http://awesome.naquadah.org/"
```

Comment:
Project home page has been changed to [https://awesomewm.org/](https://awesomewm.org/)
Will change.